api/DOMPoint.json

```diff
@@ -5,15 +5,42 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMPoint",
         "spec_url": "https://drafts.fxtf.org/geometry/#DOMPoint",
         "support": {
-          "chrome": {
-            "version_added": "61"
-          },
-          "chrome_android": {
-            "version_added": "61"
-          },
-          "edge": {
-            "version_added": "79"
-          },
+          "chrome": [
+            {
+              "version_added": "61"
+            },
+            {
+              "version_added": "2",
+              "version_removed": "39",
+              "partial_implementation": true,
+              "alternative_name": "WebKitPoint",
+              "notes": "<code>WebKitPoint</code> only supports 2D positioning using <code>x</code> and <code>y</code>."
+            }
+          ],
+          "chrome_android": [
+            {
+              "version_added": "61"
+            },
+            {
+              "version_added": "18",
+              "version_removed": "39",
+              "partial_implementation": true,
+              "alternative_name": "WebKitPoint",
+              "notes": "<code>WebKitPoint</code> only supports 2D positioning using <code>x</code> and <code>y</code>."
+            }
+          ],
+          "edge": [
+            {
+              "version_added": "79"
+            },
+            {
+              "version_added": "12",
```

Comment:
Yep http://mdn-bcd-collector.appspot.com/tests/api/WebKitPoint is supported in Edge 18.